### DISCUSSION TITLE
Multiple GoodWe Inverters

### DISCUSSION THREAD

**Discussion by jozef-moravcik** [mletenay/home-assistant-goodwe-inverter#301] [Q&A]
Hi,
I have two (soon to be three) GW10K-ET inverters controlled by a GoodWe SEC1000S Smart Controller, which also controls the export limit and distributes this value (not only this one) to the inverters according to some logic via RS485. The SEC1000S controller sets the export limit value on all connected inverters every second, so it cannot be set by another system because this value is automatically replaced. Is there any way how HA can set the export limit directly on the SEC1000S controller? I currently set this value manually in Goodwe's Promate Windows App, but I need to automate it by HA.

**Comment by nzkiwi68** [comment]
I'm not familiar with what setting you can set on the SEC1000S, but how about this for an idea?
Set the SEC1000S manually to no export limit at all using the GoodWe native tools / methods to do this.

Then you'd be free to use HA to drive the 3 x GW10K-ET inverters and write your own logic for export limit per inverter.
